app/assets/stylesheets/application.scss

```diff
@@ -1,8 +1,6 @@
 $govuk-compatibility-govuktemplate: true;
```

Comment:
We can remove this line, or set it to `false` now.

Reply:
Good spot, an entire commit is missing in the re-creation. This is has been updated. Running the diffs again to add confidence.